catalog/models.py

```diff
@@ -1,8 +1,34 @@
 import yaml
+from neo4j import GraphDatabase
+from config import config
 
 # Path to the dataset file.
 DATASET = '/catalog/components-data.yaml'
-
+    
+def get_components_neo4j():
+    """
+    Proof of concept function connects to neo4j pod and returns components.
+    """
+    # Neo4j pod credentials
+    user = config['neo4j_user']
+    pw = config['neo4j_pw']
+    url = f"bolt+s://{user}.pods.icicle.tapis.io:443"
```

Comment:
`user` cannot be the only expected url. The Pods service allows users to have non-pod_id urls.